### PR TITLE
Add the batch-status command

### PR DESCRIPTION
**Pull request by feihujiang**
Signed-off-by: feihujiang <jiangfeihu@huawei.com>

### CODE REVIEW

File: cli/sawtooth_cli/batch_status.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        Args:
+            subparsers: Add parsers to this subparser object
+            parent_parser: The parent argparse.ArgumentParser object
+    """
+    epilog = '''details:
+        Fetches the committed statuses for a single batch.
+    '''
+    parser = subparsers.add_parser(
+        'batch_status', epilog=epilog,
```
Comment (peterschwarz) [inline cli/sawtooth_cli/batch_status.py]: Change this to `batch-status`.  Operating under a no-underscores-in-command-args policy.

File: cli/sawtooth_cli/batch_status.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    parser.add_argument(
+        '--wait',
+        nargs='?',
+        const=maxsize,
+        type=int,
+        help='wait for batches to commit, set an integer to specify a timeout')
+
+    parser.add_argument(
+        'batch_id',
```
Comment (peterschwarz) [inline cli/sawtooth_cli/batch_status.py]: likewise, change this to `batch-id`

File: cli/sawtooth_cli/batch_status.py (hunk 1 above)
Comment (feihujiang) [inline cli/sawtooth_cli/batch_status.py]: Changed it.

File: cli/sawtooth_cli/batch_status.py (hunk 2 above)
Comment (feihujiang) [inline cli/sawtooth_cli/batch_status.py]: this is the position argument, should use the batch_id, not the batch-id.

**Comment by feihujiang** [comment]
@peterschwarz I updated this PR, please take a look. Thanks.

**Pull request merged**